documentation/general/dash-sonic-hld.md

```diff
@@ -359,13 +378,15 @@ priority                 = INT32 value  ; priority of the rule, lower the value,
 action                   = allow/deny
 terminating              = true/false   ; if true, stop processing further rules
 protocols                = list of INT ',' separated; E.g. 6-tcp, 17-udp; if not provided, match on all protocols
-src_addr                 = list of source ip prefixes ',' separated
-dst_addr                 = list of destination ip prefixes ',' separated
-src_port                 = list of range of source ports ',' separated
-dst_port                 = list of range of destination ports ',' separated
+src_tag                  = list of source tag name ',' separated; if not provided, match on all source TAGs or no TAG.
```

Comment:
@Pterosaur and @prsunny 
Instead of this:  '_match on all source TAGs or no TAG_'
Could we have a rewording to:  '_match on ANY tag or NO tag_'?

Per Community conversation, the intention is that if the attribute is not provided, it is like any other ACL rule.  If not provided, it will be a wildcard match.  Could we have it reworded similar to the above?

Reply:
OK, I will submit a PR to polish it.

Reply:
Change it in PR: https://github.com/sonic-net/DASH/pull/359